src/main/java/org/zaproxy/zap/extension/jwt/attacks/ClientSideAttack.java

```diff
@@ -174,6 +176,20 @@ public boolean execute() {
                     return true;
                 }
             }
+
+            // Raise an info alert if JWT is present in header.
+            HttpRequestHeader requestHeaders = msg.getRequestHeader();
+            for (HttpHeaderField headerField : requestHeaders.getHeaders()) {
+                if (headerField.getName().equals(param)) {
+                    this.raiseAlert(
+                            VulnerabilityType.FORM_PARAM,
```

Comment:
Is this purposefully form when dealing with headers?

Reply:
aah thanks for pointing it out. will correct it.